rsocket-examples/src/test/java/io/rsocket/integration/TestingStreaming.java

```diff
@@ -0,0 +1,168 @@
+package io.rsocket.integration;
+
+import io.rsocket.*;
+import io.rsocket.transport.ClientTransport;
+import io.rsocket.transport.ServerTransport;
+import io.rsocket.transport.netty.client.TcpClientTransport;
+import io.rsocket.transport.netty.server.TcpServerTransport;
+import io.rsocket.util.PayloadImpl;
+import java.time.Duration;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Supplier;
+import org.junit.Test;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+public class TestingStreaming {
+  Supplier<ServerTransport<? extends Closeable>> serverSupplier =
+      () -> {
+        //return LocalServerTransport.create("test");
+        return TcpServerTransport.create(8001);
```

Comment:
Can we avoid fixed ports?  Can you use TcpClientSetupRule to get your client and server instead?